changelog.md

```diff
@@ -1,11 +1,2 @@
-- default base docker image set to CALDP_20230208_CAL_final
-- default crds update to hst_1064.pmap
-- significant refactoring to generalize processing terms and support
-  HAP processing workflows
-  - enabled S3 output tests and made related adjustments
-  - overhauled logging, dropping Python logging for print(),  to:
-  - reduce complexity
-  - obtain log output running under pytest
-  - avoid writing to same log file from two processes concurrently: the
-    .py’s and caldp-process
-  - re-enable console output of caldp .py logs
+- default base docker image set to CALDP_cosandpin_CAL_rc1
```

Comment:
The tar change above is from v0.2.22 which was not merged because of the spec_plot issue. Could you provide a sentence to describe the above change to add to the changelog? Something like "security update for tarball extraction"?